### DISCUSSION TITLE
Add feature to chaste - Capsules in cell based

### DISCUSSION THREAD

**Discussion by Elphikaus** [Chaste/Chaste#288] [Ideas]
Hi,

I have been using Chaste as part of my PhD where I simulate capsules in cell-based chaste.

The bulk of the library section I work on is due to a workshop where some Chaste contributors wrote the bulk of the code.

I wanted to find out if there was a process to look at adding it to the main code base.

**Comment by AlexFletcher** [comment]
Hi, in practice I think the easiest way to start would be to have a chat online - if you email me (a.g.fletcher@sheffield.ac.uk), we can try to find a time when at least some Chaste developers are free, and then take it from there.

**Reply by Elphikaus** [reply]
Great. Thanks

**Reply by Elphikaus** [reply]
Email sent

**Comment by jmpf** [comment]
Added a discussion item to next hackathon (issue #287).

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: Orchestrator
- **Step**: S04
- **Link**: https://github.com/CrossTerra/monorepo/pull/2891#issuecomment-4579612505
- **Summary**: Developer completed S04 successfully (all CI checks pass); re-invoked reviewer on PR #2891 for final completion check.

**Reply by MILeach** [reply]
@Elphikaus just wanted to check that this latest comment isn't related to anything you're doing? If not we'll close this discussion

**Reply by Elphikaus** [reply]
Nothing related to me. Thanks.

**Comment by mirams** [comment]
Where did we get to with this @Elphikaus ? Still something we should try and merge in?

**Reply by Elphikaus** [reply]
Hi @mirams, I have not made any progress, lots going on. I am still keen to see it move to a PR.

**Reply by Elphikaus** [reply]
We can try and merge. Do you want me to open a PR?

**Reply by mirams** [reply]
Yeah, sounds good. We can then advise if any testing or tutorials need to be added or anything.

**Reply by Elphikaus** [reply]
Great. I think I have the code in a separate repo to my larger project. I'll try and do this over the weekend.
